<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dock  Widget Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>335</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_picture">
          <property name="text">
           <string>lab_picture</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="action_addFolder"/>
    <addaction name="action_addFile"/>
    <addaction name="action_deletePoint"/>
    <addaction name="action_ergodicPoint"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_enlarge"/>
    <addaction name="action_narrow"/>
    <addaction name="action_realSize"/>
    <addaction name="action_porperWidth"/>
    <addaction name="action_properHeight"/>
    <addaction name="action_windowFloat"/>
    <addaction name="action_windowVisible"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_addFolder"/>
   <addaction name="action_addFile"/>
   <addaction name="action_deletePoint"/>
   <addaction name="action_ergodicPoint"/>
   <addaction name="separator"/>
   <addaction name="action_enlarge"/>
   <addaction name="action_narrow"/>
   <addaction name="action_realSize"/>
   <addaction name="action_porperWidth"/>
   <addaction name="action_properHeight"/>
   <addaction name="separator"/>
   <addaction name="action_windowFloat"/>
   <addaction name="action_windowVisible"/>
   <addaction name="action_quit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_addFolder">
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_addFile">
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_deletePoint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_ergodicPoint">
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_enlarge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_narrow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_realSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>实际大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_porperWidth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>适合宽度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_properHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="toolTip">
    <string>适合高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_windowFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="toolTip">
    <string>窗口浮动</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_windowVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗口可见</string>
   </property>
   <property name="toolTip">
    <string>窗口可见</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
